### PR TITLE
Lesson09 Asssignment

### PR DESCRIPTION
**Pull request by dfspray**

### CODE REVIEW

**Comment by natasha-aleksandrova** [comment]
@dfspray can you please clarify what is this submission for? I'm seeing a folder for lesson 10 here as well and I see no submissions on the course website.

**Comment by dfspray** [comment]
@natasha-aleksandrova I wanted to just submit Lesson 09 for now. The commits to Lesson 10 just got tacked on too. Lesson10 is functional, but I thought it'd be a good idea to wait for feedback on Lesson 09 first.

**Comment by natasha-aleksandrova** [comment]
Thank you for letting me know. I wasn't sure since Lesson 9 had some code from previous PR. 
Will review Lesson 9 only for now.

File: students/Daniel_Spray/Lesson09/mailroom5.py [hunk 1]
```diff
@@ -4,70 +4,114 @@ class Donor:
...
-
-"""Establish donor data dictionary"""
-donation_data = {
-'William Gates, III': [100000.00,553784.49],
-'Mark Zuckerberg': [5000.00,5000.00,6396.10],
-'Jeff Bezos': [877.33],
-'Paul Allen': [100.00,100.00,508.42]
-}
+        """Add a list of all donors property"""
+        list = []
```
Comment (natasha-aleksandrova) [inline students/Daniel_Spray/Lesson09/mailroom5.py]: certainly _never_ use that as a variable 🙂 

this overwrites built-in `list`

File: students/Daniel_Spray/Lesson09/mailroom5.py [hunk 2]
```diff
@@ -93,89 +137,54 @@ def menu():
...
-The Charity
-""".format(**letter_dictionary)
-    print(content)
-    return(content)
-
-def create_report():
-    """Build a report"""
-    result = calculation()
-    table(result)
     return
```
Comment (natasha-aleksandrova) [inline students/Daniel_Spray/Lesson09/mailroom5.py]: no need for `return` statements in your functions.

**Pull request merged**